Sources/Subs-Post.php

```diff
@@ -1316,12 +1316,22 @@ function smtp_mail($mail_to_array, $subject, $message, $headers)
 	if (!server_parse(null, $socket, '220'))
 		return false;
 
+	// Try and determine the servers name, fall back to the mail servers if not found
+	$helo		= FALSE;
```

Comment:
Changed them over, reverted to coding standards at work where its capitals for FALSE and TRUE and lined up = if possible.